editoast/src/views/timetable.rs

```diff
@@ -933,4 +986,80 @@ mod tests {
             .json_into();
         assert_eq!(&response.error_type, "editoast:timetable:NotFound")
     }
+
+    #[rstest]
+    #[case(Some(1))]
+    #[case(None)]
+    fn train_id_parse_paced_train_exception(#[case] index: Option<u64>) {
+        let exception_key = "5062360642-f8f8-44ef-9f37-22cd0f5fcc07".to_string();
+        let paced_train_id = 50;
+
+        let paced_train_exception_id = TrainId::PacedTrainException {
+            paced_train_id,
+            exception_key: exception_key.clone(),
+            index,
+        }
+        .to_string()
+        .parse::<TrainId>()
+        .unwrap();
+        assert_eq!(
+            paced_train_exception_id,
+            TrainId::PacedTrainException {
+                paced_train_id,
+                exception_key: exception_key.clone(),
+                index
+            }
+        );
+    }
+
+    #[test]
+    fn train_id_parse_paced_train_occurrence() {
+        let index = 1;
+        let paced_train_id = 50;
+        let paced_train_occurence_id = TrainId::PacedTrainOccurrence {
+            paced_train_id,
+            index,
+        }
+        .to_string()
+        .parse::<TrainId>()
+        .unwrap();
+
+        assert_eq!(
+            paced_train_occurence_id,
+            TrainId::PacedTrainOccurrence {
+                paced_train_id,
+                index
+            }
+        );
+    }
+
+    #[test]
+    fn train_id_parse_train_schedule() {
+        assert_eq!(
+            TrainId::TrainSchedule(30)
+                .to_string()
+                .parse::<TrainId>()
+                .unwrap(),
+            TrainId::TrainSchedule(30)
+        );
+    }
+
+    #[rstest]
+    #[case("1#2#3", "Invalid PacedTrainOccurrenceId format")]
+    #[case("", "Invalid train id")]
+    #[case("#", "Invalid train id")]
+    #[case("50#", "Invalid occurence id")]
```

Comment:
```suggestion
    #[case("50#", "Invalid occurrence id")]
```
Please check that all occurrences of "occurrence" are correctly spelled.